src/lib.rs

```diff
@@ -35,10 +35,10 @@
 //! entire program terminates.
 //!
 //! Winit no longer uses a `EventLoop::poll_events() -> impl Iterator<Event>`-based event loop
-//! model, since that can't be implemented properly on web and mobile platforms and works poorly on
-//! most desktop platforms. However, this model can be re-implemented to an extent on desktops with
-//! [`EventLoopExtDesktop::run_return`]. See that method's documentation for more reasons about why
-//! it's discouraged, beyond mobile/web compatibility reasons.
+//! model, since that can't be implemented properly on some platforms (e.g web) and works poorly on
```

Comment:
I think we should mention `ios` here, since it's the only reason why it's like that. I'm not sure in what way though.

```suggestion
//! model, since that can't be implemented properly on some platforms (e.g web, ios) and works poorly on
```